### PR TITLE
Remove unnecessary NativeOp class

### PR DESCRIPTION
**Pull request by EmilyBourne**
The code contains the following message:
> The following were defined to be sympy approved nodes. If there is something
> smaller that could be used, that would be preferable. We only use them as
> tokens.

Now that sympy has been removed this is no longer necessary and a simple string can be used.

### CODE REVIEW

File: pyccel/ast/core.py [hunk 1]
```diff
@@ -38,7 +38,6 @@
 # TODO [YG, 12.03.2020]: Rename classes to avoid name clashes in pyccel/ast
 # NOTE: commented-out symbols are never used in Pyccel
```
Comment (yguclu) [inline pyccel/ast/core.py:39]: I think that this "NOTE" is no longer relevant. Please remove it

File: pyccel/ast/core.py [hunk 2]
```diff
@@ -859,23 +803,34 @@ def __init__(
...
             raise TypeError('Unrecognized Operator')
 
         self._op = op
 
         super().__init__(lhs, rhs, status, like)
 
     def __str__(self):
-        return '{0} {1}= {2}'.format(str(self.lhs), self.op._symbol,
+        return '{0} {1}= {2}'.format(str(self.lhs), self.op,
                 str(self.rhs))
```
Comment (yguclu) [inline pyccel/ast/core.py:815]: Could this fit in a single line now?

**Review by yguclu — CHANGES_REQUESTED** [review]
This looks very good to me. I have just a couple of minor comments

File: pyccel/ast/core.py (hunk 2 above)
Comment (EmilyBourne) [reply pyccel/ast/core.py]: Fixed in 908cae3

File: pyccel/ast/core.py (hunk 1 above)
Comment (EmilyBourne) [reply pyccel/ast/core.py]: Fixed in 908cae3

**Review by yguclu — APPROVED** [review]

**Pull request merged**